openraft/src/core/raft_core.rs

```diff
@@ -1278,7 +1297,14 @@ impl<C: RaftTypeConfig, N: RaftNetworkFactory<C>, S: RaftStorage<C>> RaftCore<C,
                     "send_vote_req",
                     target = display(target)
                 )),
-            );
+            )
+            .map_err(move |e| (target, e));
+
+            tasks.push(t);
+        }
+
+        while let Some(Err((target, error))) = tasks.next().await {
+            tracing::error!({error=%error, target=display(target)}, "while requesting vote");
         }
```

Comment:
Remove these lines. 

`spawn_parallel_vote_requests()` should not block until the task finishes. It should return as soon as tasks spawned.